Sprint-1/interpret/to-pounds.js

```diff
@@ -17,11 +17,17 @@ const pence = paddedPenceNumberString
 
 console.log(`£${pounds}.${pence}`);
 
-// This program takes a string representing a price in pence
+// This prograpaddedPenceNumberString takes a string representing a price in pence
 // The program then builds up a string representing the price in pounds
 
 // You need to do a step-by-step breakdown of each line in this program
 // Try and describe the purpose / rationale behind each step
 
 // To begin, we can start with
 // 1. const penceString = "399p": initialises a string variable with the value "399p"
+// 2. Line 3 Removes trailing the p
+// 3. Line 8 added 0 to front of 399 to become 0399
+// 4. Line 9 Extracts the pound portion of the value as 03 
+// 5. Line 14 Extracts the pence portion of the value as 99
```

Comment:
Is `.padEnd(2, "0")` necessary in this code? If we remove it, what value of `penceString` (a valid value) would make the code fail?

Reply:
> Is `.padEnd(2, "0")` necessary in this code? If we remove it, what value of `penceString` (a valid value) would make the code fail?

Answer 1: Removing .padEnd(2, "0")`will not have any impact because paddedPenceNumberString
  .substring(paddedPenceNumberString.length - 2) will always return the last 2 digits of penceString
Answer 2:  The code will always give a correct value from 0p to any infinite positive value.